src/m_menu.c

```diff
@@ -4165,6 +4166,9 @@ setup_menu_t gen_settings5[] = { // General Settings screen5
   {"Mouse threshold", S_NUM, m_null, M_X,
    M_Y + gen5_mouse_accel_threshold * M_SPC, {"mouse_acceleration_threshold"}},
 
+  {"Mouse smoothing", S_YESNO, m_null, M_X,
```

Comment:
Maybe we should rename it. "Short Tics Smoothing" or something like that.

Reply:
It applies to more than just shorttics though. Enabling "mouse smoothing" applies interpolation at all times when the fps is uncapped, just like before. Maybe "mouse interpolation" instead? I was trying to avoid jargon (e.g. "headphones mode" instead of "hrtf"), so "smoothing" seems simple to understand.

Reply:
My problem with "mouse smoothing" is that it sounds like an improvement, although I think our current default on most systems is smoother. I'm not sure what to call it.

Reply:
How about "raw mouse input" with it enabled by default?

Reply:
I like it.

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/1348/commits/1d7a52defd8d6d81b4650f7a0d42f65657e8d1bf